discretize/CylMesh.py

```diff
@@ -813,6 +796,7 @@ def _ishangingEz(self):
                 hangingEz = np.kron(
                     np.ones(self.nCz, dtype=bool),
                     (
+                        # True * False = False
```

Comment:
Thank you for reminding us of simple things. :) This is actually super helpful.

Reply:
:) it is!